nservicebus/azure-service-bus/dlq-forwarding.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure Service Bus DLQ Forwarding
+versions: '[7,)'
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Dead Letter Queue Forwarding
```

Comment:
I have no problem with a descriptive title on the page thb. Unless this is not a good practice, then we can change it.